docs/en/aws-elastic-serverless-forwarder.asciidoc

```diff
@@ -79,6 +79,26 @@ The forwarder can ingest logs contained within the payload of an Amazon SQS body
 
 You can set up a separate SQS queue for each type of log. The config parameter for {es} output `es_datastream_name` is mandatory. If this value is set to an {es} data stream, the type of log must be correctly defined with configuration parameters. A single configuration file can have many input sections, pointing to different SQS queues that match specific log types.
 
+[discrete]
+[[at-least-once-delivery]]
+== At-least-once delivery
+
+Elastic Serverless Forwarder ensures "at-least-once" delivery of the forwarded message.
+
+The `Continuing quque` and `Replay queue` are programmatically populated by Elastic Serverless Forwarder.
+This happens for the `Continuing quque` in the last two minutes of the lambda execution, if any message in the original trigger is left to be processed before that time (including the original triggers and the continuing and replay queues themselves).
```

Comment:
```suggestion
This happens for the `Continuing queue` in the last two minutes of the lambda execution, if any message in the original trigger is left to be processed before that time (including the original triggers and the continuing and replay queues themselves).
```
Hmm does `This` mean the creation of the queues?

Reply:
no, `This` refers to "programmatically populated"